attachment_queue/__manifest__.py

```diff
@@ -3,7 +3,7 @@
 
 {
     "name": "Attachment Queue",
-    "version": "12.0.1.0.0",
+    "version": "12.0.3.0.0",
```

Comment:
There is a conflict here, please do not specify a version here

Reply:
Corrected with a rebase, hope it's allright now